.github/workflows/tests.yml

```diff
@@ -14,37 +14,57 @@ on:
     tags: ['**']
 
 jobs:
-  test:
+  pytest:
     timeout-minutes: 20
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["3.8", "3.11"]  # Oldest and newest supported versions
-        mne-version: [mne-stable]
-
+        python-version: ["3.11"]  # Oldest and newest supported versions
+        mne-version: [mne-main]
+        qt: [PyQt6]
         include:
-          # Only test devel versions with Python 3.10
+          # Old (and PyQt5)
+          - os: ubuntu-latest
+            python-version: "3.8"
+            mne-version: mne-stable
+            qt: PyQt5
+          # PySide2
+          - os: ubuntu-latest
+            python-version: "3.10"
+            mne-version: mne-main
+            qt: PySide2
+          # PySide6
           - os: ubuntu-latest
             python-version: "3.10"
             mne-version: mne-main
+            qt: PySide6
     steps:
     - uses: actions/checkout@v3
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
+    - uses: larsoner/setup-headless-display-action@qt
```

Comment:
Just need to wait for https://github.com/pyvista/setup-headless-display-action/pull/7 to land then this can be changed to use upstream `pyvista` and this PR can be merged!